main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunnerConfig.java

```diff
@@ -63,6 +63,9 @@ public Path systemErrorsReportPath() {
   // If true, any output json will be pretty-printed.
   public abstract boolean prettyJson();
 
+  // If true, the validator will not validate the latest version.
```

Comment:
[suggestion]: This description can be read as the validator not validating rather than checking for a new version. This can be reworded like:
```suggestion
  // If true, the validator will not check for a new validator version
```